pkg/install/install.go

```diff
@@ -155,12 +155,10 @@ func (i *Installer) AdminUpgrade(ctx context.Context) error {
 		action(i.fixLBProbes),
 		action(i.fixPullSecret),
 		action(i.ensureGenevaLogging),
+		action(i.upgradeCertificates),
```

Comment:
Do we `upgradeCertificates` step here permanently or Is it going to be on-off admin update? I'm a bit worried about `LifetimeActions` getting out of sync.

Reply:
I'm hoping it's going to be one-off.  What's your concern about LifetimeActions?

Reply:
`LifetimeActions` in `CreateSignedCertificate` and in `UpgradeCertificatePolicy` should have the same values. If it's a one-off thing - let's maybe create a follow-up issue to remove the function.

If we want to keep this function - maybe let's make the connection more obvious in the code. E.g. by defining a constant and using it in both places or something like this. To make it harder to change value in one place but forget the other?

We probably not going to change it often, but we might want to reduce the time at some point.